debian/changelog

```diff
@@ -1,9 +1,15 @@
-frr (8.2~dev-1) UNRELEASED; urgency=medium
+frr (8.3~dev-1) UNRELEASED; urgency=medium
 
   * New upstream release...
 
  -- Jafar Al-Gharaibeh <jafar@atcorp.com>  Mon, 08 Nov 2021 10:00:00 +0500
 
+frr (8.2-1) unstable; urgency=medium
```

Comment:
Let us drop this until we actually  release. The release procedure includes a step to cherry pick the updated chngelog chnage from the stable 8.2 branch

Reply:
If I understand this correctly we push this as it is now and on the release date, we cherry-pick this commit to stable/8.2, and update the changelog with the real updates, instead of just "New upstream release FRR 8.2". Correct me if I'm wrong.

Reply:
Fixed.